app/src/main/java/com/duycuong/weather/utils/BindingUtils.java

```diff
@@ -34,6 +37,18 @@ public static void loadImage(ImageView imageView, int idImage) {
     public static void rounding(TextView textview, double temparature) {
         DecimalFormat df = new DecimalFormat("#");
         df.setRoundingMode(RoundingMode.CEILING);
-        textview.setText(df.format(temparature));
+        textview.setText(df.format(temparature) + (char) 0x00B0);
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.